src/bento/landingPageData.js

```diff
@@ -66,18 +66,18 @@ export const landingPageData = {
   tile3: {
     alt: '',
     img: 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/main/cds/landing/landingTile3.png',
-    titleText: 'CLICK HERE TO GET STARTED',
-    descriptionText: 'Click to Get Started or reach out to the CDS Helpdesk for any information.',
-    callToActionText: 'Start the process',
+    titleText: 'SUBMIT DATA',
+    descriptionText: 'To get started with the submission please complete the online questionnaire.',
+    callToActionText: 'CDS HELPDESK',
     callToActionLink: '/datasubmit', // Link to the "Submit" Static Page
```

Comment:
Please refer to these two bullet points within the ticket. 

- A "SUBMIT DATA" section, which contains the following text "To get started with the submission please complete the online questionnaire." and should link to https://dataservice.datacommons.cancer.gov/#/datasubmit   Not passed Please implement
- A "CDS HELPDESK" link associated with the CDS Helpdesk email (cdshelpdesk@mail.nih.gov)  Not passed Please implement

The "SUBMIT DATA" title should link to the /datasubmit URL
The "CDS HELPDESK" button should link to the e-mail popup

Current implementation is reversed. Please update the references
 - src/bento/landingPageData.js - line 72 (update to email mailto)
 - src/pages/landing/landingView.js - line 132 (update href to /datasubmit page)